includes/admin/services/class-rop-pinterest-service.php

```diff
@@ -438,14 +407,22 @@ public function share( $post_details, $args = array() ) {
 		}
 
 		// Don't shorten post link, pinterest might reject post if shortened and it also looks bad on pinterest with a shortlink
-		$pin = $api->pins->create(
-			array(
+		try {
+
+			$info_to_pin = array(
 				'note'      => $this->strip_excess_blank_lines( $post_details['content'] ) . $post_details['hashtags'],
 				'image_url' => $post_details['post_image'],
 				'board'     => $args['id'],
-				'link'      => $post_details['post_url'],
```

Comment:
@mihaiiro I think you removed the `link` parameter by accident here, or was it intentional?

Reply:
I think I removed it when I first started to work on the Pinterest API. For some reason, it didn't work with it. 

Added the parameter back and tested it. It seems to be working just fine https://d.pr/i/8zoITb